webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const webpack = require('webpack');
+
+module.exports = {
+  entry: {
+    common: './js/src/common.js',
+    documentation: './js/src/documentation.js',
+    install: './js/src/install.js',
+    nightly: './js/src/nightly.js',
+    search: './js/src/search.js',
```

Comment:
Hey @Daniel15, thanks for the suggestion. I've never done code splitting before, so now I manually did what code would be split too, with a hand-made common.js that hosts things that all pages have. Could you tell me how that would be set up properly? Thanks!